Add Closure to CSV Download

TaiWilkin opened the pull request:
## Overview

Add the facility closure status to the CSV downloads in the case that
the `report_a_facility` switch is on.

Connects #1248 

## Demo

<img width="753" alt="Screen Shot 2021-02-17 at 3 10 53 PM" src="https://user-images.githubusercontent.com/21046714/108271387-bb58c480-713e-11eb-973b-f123716c5240.png">

## Testing Instructions

* Run `./scripts/server`
* Turn off the Reports switch and download the facilities CSV
    - [ ] It should not have an 'is_closed' column
* Turn on the Reports switch and download the facilities CSV
    - [ ] It should have an 'is_closed' column with appropriate data

## Checklist

- [ ] `fixup!` commits have been squashed
- [ ] CI passes after rebase
- [ ] CHANGELOG.md updated with summary of features or fixes, following [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) guidelines

jwalgran commented on src/app/src/actions/logDownload.js:
```diff
@@ -38,6 +38,7 @@ export function logDownload() {
 
             const vectorTileFlagIsActive = get(featureFlags, 'flags.vector_tile', false);
             const ppeIsActive = get(featureFlags, 'flags.ppe', false);
+            const reportsAreActive = get(featureFlags, 'flags.report_a_facility', false);
```
👍 Nice integration with our new switch

jwalgran reviewed (approved):
Nice, clear, correct implementation. I like seeing a new JS test.

While this does match the spec of the feature as listed in the issue, when I take a look at the CSV after closing a facility and reopening one I see an opportunity to make this a little friendlier to less technical users

<img width="140" alt="Screen Shot 2021-02-17 at 8 14 05 PM" src="https://user-images.githubusercontent.com/17363/108300050-d5090480-715c-11eb-8636-1b9f735a381f.png">

An alternate formatting I came up with is to output the string "CLOSED" in the `is_closed` column if `Facility.is_closed` is `True`, otherwise leave the column blank. This reduces the number of possible states from 3 to 2 and avoids some of the "double negativeness" of having True representing closed. I'm interested in what you think about this alternate formatting, if you have another suggestion, or think it is more appropriate to keep it as is currently implemented.

TaiWilkin commented:
I agree that 'CLOSED' is going to be clearer for non-technical users. I've updated the code to use 'CLOSED' when True and otherwise display nothing, and updated the test to check for handling both True and False.
